content/en/lotus/lotus-config/configuration.md

```diff
@@ -0,0 +1,273 @@
+  #SimultaneousTransfersForRetrieval = 20
+
+
+[Wallet]
+  # type: string
+  # env var: LOTUS_WALLET_REMOTEBACKEND
+  #RemoteBackend = ""
+
+  # type: bool
+  # env var: LOTUS_WALLET_ENABLELEDGER
+  #EnableLedger = false
+
+  # type: bool
+  # env var: LOTUS_WALLET_DISABLELOCAL
+  #DisableLocal = false
+
+
+[Fees]
+  # type: types.FIL
+  # env var: LOTUS_FEES_DEFAULTMAXFEE
+  #DefaultMaxFee = "0.07 FIL"
+
+
+[Chainstore]
+  # type: bool
+  # env var: LOTUS_CHAINSTORE_ENABLESPLITSTORE
+  #EnableSplitstore = false
+
+  [Chainstore.Splitstore]
+    # ColdStoreType specifies the type of the coldstore.
+    # It can be "universal" (default) or "discard" for discarding cold blocks.
+    #
+    # type: string
+    # env var: LOTUS_CHAINSTORE_SPLITSTORE_COLDSTORETYPE
+    #ColdStoreType = "universal"
+
+    # HotStoreType specifies the type of the hotstore.
+    # Only currently supported value is "badger".
+    #
+    # type: string
+    # env var: LOTUS_CHAINSTORE_SPLITSTORE_HOTSTORETYPE
+    #HotStoreType = "badger"
+
+    # MarkSetType specifies the type of the markset.
+    # It can be "map" (default) for in memory marking or "badger" for on-disk marking.
+    #
+    # type: string
+    # env var: LOTUS_CHAINSTORE_SPLITSTORE_MARKSETTYPE
+    #MarkSetType = "map"
+
+    # HotStoreMessageRetention specifies the retention policy for messages, in finalities beyond
+    # the compaction boundary; default is 0.
+    #
+    # type: uint64
+    # env var: LOTUS_CHAINSTORE_SPLITSTORE_HOTSTOREMESSAGERETENTION
+    #HotStoreMessageRetention = 0
+
+    # HotStoreFullGCFrequency specifies how often to perform a full (moving) GC on the hotstore.
+    # A value of 0 disables, while a value 1 will do full GC in every compaction.
+    # Default is 20 (about once a week).
+    #
+    # type: uint64
+    # env var: LOTUS_CHAINSTORE_SPLITSTORE_HOTSTOREFULLGCFREQUENCY
+    #HotStoreFullGCFrequency = 20
+```
+
+## Connectivity
+
+Usually your lotus daemon will establish connectivity with others in the network and try to make itself diallable using uPnP. If you wish to manually ensure that your daemon is reachable:
+
+- Set a fixed port of your choice in the `ListenAddresses` in the Libp2p section (i.e. 6665).
+- Open a port in your router that is forwarded to this port. This is usually called featured as "Port forwarding" and the instructions differ from router model to model but there are many guides online.
+- Add your public IP/port to `AnnounceAddresses`. i.e. `/ip4/<yourIP>/tcp/6665`.
+
+Note that **it is not a requirement to use Lotus as a client to the network to be fully reachable**, as your node already dials-out to others nodes and miners directly.
+
+## Environment variables
+
+Some aspects of the Lotus Node can be controlled using environment variables.
+
+Variables common to most Lotus binaries:
+
+- `LOTUS_FD_MAX`: Sets the file descriptor limit for the process
+- `LOTUS_JAEGER`: Sets the Jaeger URL to send traces. See TODO.
```

Comment:
> See TODO.

Where's this meant to go?

Reply:
It is carried forwarded from older content. I did not add it. Shall we remove it?
I think someone wanted to add Jaeger URL maybe?

Reply:
Yeah let's remove it for now. Thanks.

Reply:
I have filed the latest commit to remove this TODO. I have also hidden the blog and empty miner tutorials. Reload your browser from source to avoid cached version and you should see the changes.